_includes/ea/syllabus.html

```diff
@@ -0,0 +1,8 @@
+<h3 id="syllabus-shell"></h3>
+<ul>
+  <li>Getting Started with Conda</li>
+  <li>Working with Environments</li>
+  <li>Using Packages and Channels</li>
+  <li>Sharing Environments</li>
+  <li><a href="https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists">Reference...</a></li>
```

Comment:
I think its fine to add the reference, because both the Windows and Linux instructions for setting up the environment seem to be okay, but we might wait for confirmation of the OSX set up instructions. 

If people accidentally follow those instructions then it should still be fine. 
But perhaps I am missing another reason why we do not want to include this reference

Reply:
The list items do not seem to turn into hyperlinks, I am not sure why or how ...

Reply:
Lets ask mateusz

Reply:
As discussed with Mateusz, I removed the reference link.